mono/metadata/native-library.c

```diff
@@ -1456,16 +1456,6 @@ pinvoke_probe_for_symbol (MonoDl *module, MonoMethodPInvoke *piinfo, const char
 
 	g_assert (error_msg_out);
 
-#ifdef HOST_WIN32
-	if (import && import [0] == '#' && isdigit (import [1])) {
```

Comment:
I believe the intent was probably to check if its an ordinal and then use the string version of the ordinal in the import, but that would never work due to how GetProcAddress lpProcName parameter works:

`The function or variable name, or the function's ordinal value. If this parameter is an ordinal value, it must be in the low-order word; the high-order word must be zero.`